src/latexify/codegen/function_codegen.py

```diff
@@ -355,6 +355,9 @@ def visit_Call(self, node: ast.Call) -> str:
             (r"\mathrm{" + func_str + r"}\left(", r"\right)"),
         )
 
+        if func_str == "ndarray" or func_str == "array":
+            return self._generate_matrix(node)
```

Comment:
I think we need to fall back to the default codegen if _generate_matric failed for some reason.
And it would be better to return None if _generate_matrix detected unsupported syntax, because raising the exception destroyes the entire process.
```suggestion
            latex = self._generate_matrix(node)
            if latex is not None:
                return latex
```